canopen_core/include/canopen_core/node_interfaces/node_canopen_master.hpp

```diff
@@ -85,7 +85,7 @@ class NodeCanopenMaster : public NodeCanopenMasterInterface
   std::string master_dcf_;
   std::string master_bin_;
   std::string can_interface_name_;
-  uint32_t timeout_;
+  uint32_t timeout_ = 2000;
```

Comment:
Single source of truth. The logging and the actual default were already out of date.